### ISSUE TITLE
Cannot deserialize instance of `com.crowdin.client.screenshots.model.Tag`

### ISSUE THREAD

**Issue by kptfh** [crowdin/crowdin-api-client-java#87]
While trying to call 
`public ResponseObject<Tag> addTag(Long projectId, Long screenshotId, List<AddTagRequest> request)`

Get the following error
```
HttpException(error=HttpException.Error(code=null, message=Cannot deserialize instance of `com.crowdin.client.screenshots.model.Tag` out of START_ARRAY token
 at [Source: (String)"{"data":[{"data":{"id":17,"screenshotId":19,"stringId":31649,"position":null,"createdAt":"2022-06-20T10:59:52+00:00"}},{"data":{"id":19,"screenshotId":19,"stringId":31651,"position":null,"createdAt":"2022-06-20T10:59:52+00:00"}},{"data":{"id":21,"screenshotId":19,"stringId":31653,"position":null,"createdAt":"2022-06-20T10:59:52+00:00"}},{"data":{"id":23,"screenshotId":19,"stringId":31655,"position":null,"createdAt":"2022-06-20T10:59:52+00:00"}},{"data":{"id":25,"screenshotId":19,"stringId":31657"[truncated 206 chars]; line: 1, column: 9] (through reference chain: com.crowdin.client.screenshots.model.TagResponseObject["data"])))
	at com.crowdin.client.core.http.exceptions.HttpException.fromMessage(HttpException.java:22)
```

Server returns List of tags while Java client expect only one tag in response !

**Comment by andrii-bodnar** [comment]
Hi @kptfh!

We had a fix for a similar issue in the release [1.3.19](https://github.com/crowdin/crowdin-api-client-java/releases/tag/1.3.19). Are you using the latest version of this API Client?

**Comment by kptfh** [comment]
I've checked the code in master branch and I see that error still there
